config-model/src/main/java/com/yahoo/vespa/model/container/Identity.java

```diff
@@ -1,23 +1,25 @@
 // Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package com.yahoo.vespa.model.container;
 
+import com.yahoo.container.core.identity.IdentityConfig;
+import com.yahoo.vespa.model.container.component.SimpleComponent;
+
 /**
  * @author mortent
  */
-public class Identity {
+public class Identity extends SimpleComponent implements IdentityConfig.Producer {
     private final String domain;
     private final String service;
 
     public Identity(String domain, String service) {
+        super("com.yahoo.container.jdisc.athenz.impl.AthenzIdentityProviderImpl");
```

Comment:
If the class is available on the classpath, use `com.yahoo.container.jdisc.athenz.impl.AthenzIdentityProviderImpl.class.getName()` instead